core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -81,15 +80,15 @@ public boolean isGameLost() throws IOException {
         for (PlayerBase player : players) {
             if (player.isDead) {
                 System.out.println("Game lost!");
-                writeToFile("C:\\Outputs.txt","Game lost!");
+                writeToFile("C:\\Outputs.txt", "Game lost!");
```

Comment:
here I guess we need to send a variable for the path...for example I use a mac and I don't have disc C.